### PR TITLE
#16817 2.0 Chinese mkdocs-cn.yml

### PR DESCRIPTION
**Pull request by alaahong**
Draft the 2.0 Chinese mkdocs-cn.yml

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [X] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/appium/docs/mkdocs-cn.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+INHERIT: ./base-mkdocs.yml
+site_url: https://appium.github.io/appium/docs/en
```
Comment (jlipps) [inline packages/appium/docs/mkdocs-cn.yml:2]: this (and the line below) should probably reference `cn` rather than `en`, right?

Comment (alaahong) [reply packages/appium/docs/mkdocs-cn.yml]: Thanks for pointing that, try to local build, met below exception
`jinja2.exceptions.TemplateNotFound: partials/languages/cn.html`
Just search from mkdocs, seems they won't support such word, so change as zh and reformat all folder and resources.
https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language

**Comment by alaahong** [comment]
Just update all cn resources as zh which is following mkdocs internal setting.

**Review by jlipps — COMMENTED** [review]
this all looks good however I'd prefer not to copy over the english text and just add chinese text when it's translated. if you want to have TODOs in chinese that's fine with me, or we can just limit the TOC to what's already translated?

**Comment by alaahong** [comment]
> this all looks good however I'd prefer not to copy over the english text and just add chinese text when it's translated. if you want to have TODOs in chinese that's fine with me, or we can just limit the TOC to what's already translated?

Well, just revert the content as TODO. Prefer to list all in TOC.

BTW, do we have the possible to add any tag or flag to mention other language to catch up with original English content? Otherwise, translator might be missed the work once almost pages translated but few pieces updated.

**Comment by jlipps** [comment]
OK I created @appium/documentors that we can ping on docs updates. Any translators can be added to that group.

**Review by jlipps — APPROVED** [review]
LGTM now, will merge when the package-lock conflict is fixed

**Comment by alaahong** [comment]
> OK I created @appium/documentors that we can ping on docs updates. Any translators can be added to that group.

Thanks for your invitation. 

For this PR, just merge latest code and accept conflict from 2.0 branch, is that fine? Seems caused by I add zh-doc command into package.json

**Comment by jlipps** [comment]
You'll need to somehow resolve the package.json conflicts then. If all you did is add a line it should be fairly simple to figure out how to make the change? Unfortunately I cannot rebase with a conflict present.

**Comment by alaahong** [comment]
@jlipps  just re-fork the project, then manually copy changes back to branch.
But seems today's build always failed (there are two build failed, one is npm install, another is passed 14.x and 16.x, but failed 18.x), should no impact by such md files changed. Can you help to retrigger once you saw this?

Thanks.

**Comment by jlipps** [comment]
yeah failures seem unrelated so i went ahead and merged it. thanks so much @alaahong !

**Pull request merged**